### ISSUE TITLE
Test setup fails when external libraries are used

### ISSUE THREAD

**Issue by realisation** [dapphub/dapptools#426]
Whenever I use an external library, my tests fail at the setup phase.

```
[FAIL] setUp()

Failure: setUp
 
  src/test/deposits/suiteOne.t.sol:SelectiveDepositSuiteOne
   ├╴constructor
   └╴initialize test
      ├╴create <unknown contract> <no source map>
      │  └╴← 0 bytes of code
      ├╴create <unknown contract> <no source map>
          └╴← 0 bytes of code
```

Those calls are the creation of mock ERC20 contracts, interesting that they return 0 bytes.

More worrisome though is that I can't test my code in the state it will use for production. 

Is this a problem with linking libraries? Will it pose more problems when it is time to deploy?

**Comment by MrChico** [comment]
Do you have a setUp function defined?

**Comment by realisation** [comment]
Yeah, the problem above arises when I uncomment lines in the contract I am testing that call external functions from a library I made. If I recomment them, or make those functions internal, the problem goes away.

Those logs are the creation of the ERC20 mocks that occur by calling a function inside the setup function of `suiteOne.t.sol`

**Comment by realisation** [comment]
I was able to reproduce the issue here - https://github.com/realisation/dapp-test-external-library-example

Its a bare bones contract with bare bones external lib showing how `dapp test` fails in the same manner reported above

**Comment by MrChico** [comment]
Thanks for the example @realisation, I have a fix for you to try: #437

**Comment by MrChico** [comment]
Fixed by #437
